### PR TITLE
TEL-4313 Upgrade Zone Health to Play 3.0

### PR DESCRIPTION
**Pull request by jonnydh**
Followed Platops migration guide

### CODE REVIEW

**Review by gavD — DISMISSED** [review]

File: project/plugins.sbt [hunk 1]
```diff
@@ -2,6 +2,6 @@ resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefac
 resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"     % "3.3.0")
-addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables" % "2.1.0")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"         % "2.8.7")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"     % "3.20.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables" % "2.5.0")
+addSbtPlugin("org.playframework" % "sbt-plugin"         % "3.0.1")
```
Comment (ma3574) [inline project/plugins.sbt:7]:
```suggestion
addSbtPlugin("org.playframework" % "sbt-plugin"         % "3.0.1")
addSbtPlugin("com.timushev.sbt"  % "sbt-updates"        % "0.6.4")
```

Worth adding the updates plugin too

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -4,27 +4,27 @@ import sbt._
 
 object AppDependencies {
 
-  val hmrcBootstrapVersion = "5.7.0"
-  val hmrcMongoVersion     = "0.51.0"
+  val hmrcBootstrapVersion = "8.4.0"
+  val hmrcMongoVersion     = "1.7.0"
+  val mockitoVersion       = "1.17.29"
```
Comment (ma3574) [inline project/AppDependencies.scala:9]: Worth upping this to `1.17.30`

Comment (ma3574) [inline project/AppDependencies.scala]:
```suggestion
  val mockitoVersion       = "1.17.30"
```

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -4,27 +4,27 @@ import sbt._
...
-    "org.mockito"            %% "mockito-scala-scalatest"  % "1.16.23"            % Test
+    "org.mockito"            %% "mockito-scala-scalatest"  % mockitoVersion       % Test
   )
 
   val itTest = Seq(
-    "uk.gov.hmrc"            %% "bootstrap-test-play-28"   % hmrcBootstrapVersion % "it",
-    "uk.gov.hmrc.mongo"      %% "hmrc-mongo-test-play-28"  % hmrcMongoVersion     % "it",
+    "uk.gov.hmrc"            %% "bootstrap-test-play-30"   % hmrcBootstrapVersion % "it",
+    "uk.gov.hmrc.mongo"      %% "hmrc-mongo-test-play-30"  % hmrcMongoVersion     % "it",
     "com.vladsch.flexmark"   %  "flexmark-all"             % "0.35.10"            % "it",
```
Comment (ma3574) [inline project/AppDependencies.scala]:
```suggestion
    "com.vladsch.flexmark"   %  "flexmark-all"             % "0.64.8"             % "it",
```

**Review by gavD — APPROVED** [review]

**Pull request merged**